pkg/wrapper/nmt_wrapper_test.go

```diff
@@ -1,4 +1,4 @@
-package wrapper
+package wrapper_test
```

Comment:
👍 I like the `_test` pattern b/c it forces tests to be written against the public API

Reply:
totally. It's generally my default. Only when the public functions become very difficult to test would I result to testing the constituent private methods